### PR TITLE
Apply the Clumit theme to `SelectComplex` and `Modal` components

### LINKED ISSUE #117: Detection menu Clumit 디자인 적용
해당 컴포넌트: `SelectComplex`, `Modal`

운영관리 화면에서는 아래 스크린샷처럼 나오는데 
Screenshot 2024-07-05 at 8 52 18 AM

Detection 화면에서 버튼이 2개일때 아래 처럼 나와서 Modal 쪽 수정이 필요합니다.
Screenshot 2024-07-05 at 8 50 58 AM

### PR DESCRIPTION
**Pull request by dayeon5470**
Fixes #117

### CODE REVIEW

**Comment by sehkone** [comment]
It looks necessary to rebase this.

**Review by sehkone — DISMISSED** [review]

**Comment by sophie-cluml** [comment]
@dayeon5470 commit message의 동사를 "Applied" -> "Apply" 로 변경 부탁드립니다. https://cbea.ms/git-commit/#imperative

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added `cc-password` feature in `Cargo.toml`.
 - Applied Clumit theme to `Sort`, `SelectMini`, `SelectSearchable`, `Input`,
   `view_asterisk`, `Notification`.
+- Applied Clumit theme to `WholeList`, `Tag`, and `HostNetworkHtml`.
+- Applied Clumit theme to `Modal` and `SelectComplex`.
```
Comment (sehkone) [inline CHANGELOG.md:20]: These two lines should be mentioned in the `Unreleased` section, not in the previous version.

Comment (sehkone) [reply CHANGELOG.md:20]: @dayeon5470 

Could you handle the above and rebase this?

Comment (dayeon5470) [reply CHANGELOG.md]: 수정 완료

File: src/select/complex/view.rs [hunk 2]
```diff
@@ -481,7 +514,21 @@ impl Model {
...
+                                            let top_width = if cfg!(feature = "pumpkin-dark") {
+                                                90
+                                            } else {
+                                                70
+                                            };
+                                            let list_top = if cfg!(feature = "pumpkin-dark") {
+                                                42
+                                            } else {
+                                                28
+                                            };
```
Comment (sehkone) [inline src/select/complex/view.rs:531]: I think the following would be simpler:

```rust
let (top_bg_color, top_width, list_top) = if cfg!(feature = "pumpkin-dark") {
...
```

File: src/select/complex/view.rs [hunk 3]
```diff
@@ -27,12 +40,24 @@ impl Model {
...
+            if cfg!(feature = "pumpkin-dark") {
+                "background-image: url('/frontary/clumit-collapse-contents.png');"
+            } else {
+                "background-image: url('/frontary/collapse-contents.png');"
+            }
+        } else if cfg!(feature = "pumpkin-dark") {
+            "background-image: url('/frontary/clumit-expand-contents.png');"
         } else {
             "background-image: url('/frontary/expand-contents.png');"
         };
```
Comment (sehkone) [inline src/select/complex/view.rs:63]: I think it woud be better to simplify the repetition of the same structure like:

```rust
let (style_list, style_input) = if self.view_list {
...
```

File: src/select/complex/view.rs [hunk 4]
```diff
@@ -13,11 +13,24 @@ use crate::{
...
+                    "background-image: url('/frontary/radio-opener-checked.png');"
+                }
+            }
+            _ => {
+                if cfg!(feature = "pumpkin-dark") {
+                    "background-image: url('/frontary/clumit-radio-opener-unchecked.png');"
+                } else {
+                    "background-image: url('/frontary/radio-opener-unchecked.png');"
+                }
+            }
```
Comment (sehkone) [inline src/select/complex/view.rs:33]: I suggest the following:

```rust
        let postfix = match self.check_status(ctx, false) {
            CheckStatus::Checked => "checked",
            _ => "unchecked",
        };
        let style_all = if cfg!(feature = "pumpkin-dark") {
            format!("background-image: url('/frontary/clumit-radio-opener-{postfix}.png');",)
        } else {
            format!("background-image: url('/frontary/radio-opener-{postfix}.png');",)
        };
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,6 +10,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 
 - Applied Clumit theme to `WholeList`, `Tag`, and `HostNetworkHtml`.
+- Applied Clumit theme to `Modal` and `SelectComplex`.
```
Comment (sehkone) [inline CHANGELOG.md:13]: These two lines can be combined into just one line.

File: src/select/complex/view.rs (hunk 2 above)
Comment (dayeon5470) [reply src/select/complex/view.rs]: 수정 완료

File: src/select/complex/view.rs (hunk 3 above)
Comment (dayeon5470) [reply src/select/complex/view.rs]: 수정 완료

File: src/select/complex/view.rs (hunk 4 above)
Comment (dayeon5470) [reply src/select/complex/view.rs]: 수정 완료

File: CHANGELOG.md (hunk 5 above)
Comment (dayeon5470) [reply CHANGELOG.md]: 수정 완료

**Review by sophie-cluml — APPROVED** [review]
LGTM

**Review by sehkone — APPROVED** [review]

**Pull request merged**